### PR TITLE
[CI:ALL] Bump to v5.2.0-rc2

### PR DESCRIPTION
**Pull request by mheon**
Now with 100% more release notes.

Main has been merged into the branch, but this will be the last time - all further changes in v5.2 will be via cherry-pick.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL

**Review by giuseppe — APPROVED** [review]
LGTM

File: RELEASE_NOTES.md
Comment (Luap99) [inline RELEASE_NOTES.md:17]: this should point out that that means virtiofsd must be installed on the host and there must be symlink created to the proper location, ref https://github.com/containers/podman/commit/180cc6f86324161c641513f291a1d01434876518

Comment (Luap99) [reply RELEASE_NOTES.md:17]: (but I guess I am fine to leave it for now and add it for the final release or next rc)

**Comment by Luap99** [comment]
/lgtm
/hold

Adding hold so you can merge it at your convenience and push the tag directly after it.

**Comment by mheon** [comment]
/hold cancel
TY

**Pull request merged**
